mkdocs.yml

```diff
@@ -100,3 +107,9 @@ nav:
       - Conventions: reference/python/conventions.md
       - Virtual Environments: reference/python/virtual-environments.md
     - Robot Operating System: reference/ros.md
+    - Sailing Knowledge:
+      - reference/sailing/index.md
+      - Parts of a Sailboat: reference/sailing/boat_parts.md
```

Comment:
This will make the dropdown itself a page. Could you change the config file to the following:

```yaml
- Sailing Knowledge:
   - Overview:
     -  reference/sailing/index.md
   - Parts of a Sailboat: reference/sailing/boat_parts.md
   ...
```

This will make the overview its own page while avoiding dropdowns to be pages.

Reply:
- For some background, see #46

Reply:
Sounds good, I made the requested changes.